profiler/README.md

```diff
@@ -1,9 +1,46 @@
 
-The profiler is in its infancy and is disabled by default.
+🚨 The profiler is in beta and its use is currently restricted.
 
-It should be considered experimental and is completely unsupported.
+# About
 
-# configuration
+The `splunk-otel-java` agent includes an "always-on" profiler that can be enabled with a configuration
+setting. This profiler periodically captures the call stack state for all JVM threads and
+saves these to the Splunk Observability Cloud. Users can then view a flamegraph of application
+call stacks and inspect individual code-level call stacks for relevant traces.
+
+## How?
+
+The profiler leverages the JVM's Java Flight Recorder (JFR) to perform periodic call
+stack sampling. Every recording period, a new JFR recording file is flushed to disk and then
+the events are replayed.
+
+In order to associate `jdk.ThreadDump` events from JFR with OpenTelemetry spans, a custom
+OpenTelemetry `ContextStorage` implementation is used to synthesize `otel.ContextAttached`
+events each time a span context change occurs. These context changes are then tracked
+during replay so that a call stack can be associated with the appropriate span.
+
+Stack trace data is embedded as a string inside of an OTLP logs payload. The
+[Splunk OpenTelemetry Connector](https://github.com/signalfx/splunk-otel-collector)
+will detect this profiling data inside of OTLP logs and will help it along
+its ingest path.
```

Comment:
```suggestion
To associate `jdk.ThreadDump` events from JFR with OpenTelemetry spans, a custom
OpenTelemetry `ContextStorage` implementation generates `otel.ContextAttached`
events each time a span context change occurs. These context changes are then tracked
during replay so that a call stack can be associated with the appropriate span.

Stack trace data is embedded as a string inside of an OTLP logs payload. The
[Splunk OpenTelemetry Connector](https://github.com/signalfx/splunk-otel-collector) detects
this profiling data inside of OTLP logs and help it along its ingest path.
```

What do we mean by "help it along its ingest path" technically?